### PR TITLE
Feature/3 implement basics css styling

### LINKED ISSUE #3: Implement basics CSS Styling
### Feature Description
This feature involves implementing basic CSS styling for various components on the website.

### Estimated Effort
Medium

### Tasks

- [x] decide which CSS-framework we should use
- [x] implement it in the project so we can use it

### Acceptance criteria
CSS can be used by the developers

### PR DESCRIPTION
**Pull request by KeiraShadow**
Please check if this is ok. I will have to modify this constantly

### CODE REVIEW

File: .vscode/settings.json
Comment (devidda) [inline .vscode/settings.json:1]: Needs to be inserted into .gitignore

File: src/main.css
Comment (devidda) [inline src/main.css:1]: Needs to be deleted, redundant to src/lib/main.css

**Review by devidda — CHANGES_REQUESTED** [review]
Some small comments, is it correct that the menu-button in the top right does not work? (I could only test it with mobile yet)

File: src/main.css
Comment (KeiraShadow) [reply src/main.css:1]: I deleted the unnecessary parts and I'm unsure which menu Button you mean? Btw I need to remove the container around the under the menu. it makes it look really weird

**Review by devidda — APPROVED** [review]

**Pull request merged**